atomic_reactor/plugins/pre_resolve_module_compose.py

```diff
@@ -25,8 +25,11 @@
 
 import os
 import re
-from modulemd import ModuleMetadata
-from pdc_client import PDCClient
+try:
+    from modulemd import ModuleMetadata
+    from pdc_client import PDCClient
```

Comment:
is this just to avoid ` logger.warning("can't load module '%s': %r", f, ex)` in plugin.py or does it cause a failure somewhere? I'm basically OK with this, though it's a bit ugly. This approach does seem to be inconsistent with the handling of dockpulp, where the warning is left.

Reply:
Agree, this change is not needed, removed